core/blockchain.go

```diff
@@ -2052,6 +2067,10 @@ func (bc *BlockChain) insertChain(chain types.Blocks, verifySeals bool) (int, er
 			lastCanon = block
 			continue
 		}
+
+		if bc.verifyManager != nil {
+			bc.verifyManager.NewBlockVerifyTask(block.Header())
```

Comment:
New verify task after the block has been inserted into the chain, not here. After bc.writeBlockWithState(block, receipts, logs, statedb, false)

Reply:
> New verify task after the block has been inserted into the chain, not here. After bc.writeBlockWithState(block, receipts, logs, statedb, false)

done